cmd/check-balance/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/4chain-AG/spv-wallet-regression/internal/utils"
+	"github.com/4chain-AG/spv-wallet-regression/internal/wallet"
+)
+
+const (
+	minimalBalance = 20
```

Comment:
Just a thought:
- Maybe introduce this "minimal balance" as reusable variable between both:
-> operator,
-> check-balance.

Because if we add some tests or modify existing we need to change this value in two different places 🤔